flink-table/flink-table-common/src/main/java/org/apache/flink/table/api/dataview/MapView.java

```diff
@@ -58,15 +61,15 @@
  *   public MapView<String, Integer> map = new MapView<>();
  *
  *   // or explicit:
- *   // {@literal @}DataTypeHint("MAP<STRING, INT>")
+ *   // @DataTypeHint("MAP < STRING, INT >")
  *   // public MapView<String, Integer> map = new MapView<>();
```

Comment:
remove `// public MapView<String, Integer> map = new MapView<>();`

Reply:
no, this is intentional. otherwise two lists would exist in the accumulator

Reply:
@twalthr it seems a duplicate of line 58 " *   public MapView<String, Integer> map = new MapView<>();".